docs/self-managed/zeebe-deployment/kubernetes/index.md

```diff
@@ -4,23 +4,24 @@ title: "Kubernetes deployment"
 sidebar_label: "Overview"
 ---
 
-We recommend you use Kubernetes when deploying Zeebe to production.
+We recommend you use Kubernetes when deploying Camunda Cloud Self-Managed to production.
 
-## Deployment options
+This chapter is divided into the following sections:
 
-| Option                                                      | Comment                                                                                                                                                                                     |
-| ----------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| [Helm](helm/index.md)                                       | Helm allows you to choose exactly what chart (set of components) you want to install and how these components need to be configured.                                                    |
+- [General information](index.md#general-information)
+- [Prerequisites](index.md#prerequisites)
+- [Getting to know and installing Camunda Cloud Helm charts](helm/installing-helm)
+- [Accessing Operate from outside a Kubernetes cluster](helm/accessing-operate-tasklist)
 
 ## General information
 
 ### Broker
 
-Zeebe broker nodes need to be deployed as a [StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) to preserve the identity of cluster nodes. StatefulSets require persistent storage, which must be allocated in advance. Depending on your cloud provider, the persistent storage differs as it is provider-specific.
+Camunda Cloud broker nodes need to be deployed as a [StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) to preserve the identity of cluster nodes. StatefulSets require persistent storage, which must be allocated in advance. Depending on your cloud provider, the persistent storage differs as it is provider-specific.
 
-At [helm.camunda.io](https://helm.camunda.io/), you'll find helm charts to configure a 3-broker cluster with the Elasticsearch exporter and Operate.
+At [helm.camunda.io](https://helm.camunda.io/), you'll find a Helm chart to configure a three-broker cluster with the Elasticsearch exporter and Operate. This size is comparable with the Production-S cluster plan in Camunda Cloud SaaS. It should be sufficient for 80% of use cases.
```

Comment:
:wrench: We should also mention elasticsearch and tasklist deployment

Reply:
Thanks for your very fast review @Zelldon! I will try to resolve these suggestions on Monday 👍

Reply:
How about this sentence instead: "...you'll find a Helm chart to configure a two-broker cluster with two Elasticsearch instances, Operate, the Zeebe Gateway and Tasklist"?

Reply:
Sorry but it is three broker :) otherthan that sounds right